components/aws/sagemaker/tests/unit_tests/tests/test_utils.py

```diff
@@ -42,4 +42,59 @@ def test_cw_logging_error(self):
 
         with patch('logging.Logger.error') as errorLog:
             _utils.print_logs_for_job(mock_cw_client, '/aws/sagemaker/FakeJobs', 'fake_job_name')
-            errorLog.assert_called()
\ No newline at end of file
+            errorLog.assert_called()
+
+    def test_stop_training_job(self):
```

Comment:
Hey can you move these out to their individual existing test files 

this "test_utils.py" is meant for testing functions which are used by all the components
eg: "_utils.stop_training_job()" is used only by training job so test that in "test_train.py"
"_utils.print_logs_for_job()" is used in all the components so we test that here

Reply:
Agree.

Perhaps this file should be renamed to `test_common.py` then? Or something a little less ambiguous?